lib/go-atscfg/parentdotconfig.go

```diff
@@ -839,7 +865,11 @@ func getParentDSParams(ds DeliveryService, profileParentConfigParams map[string]
 	if !ok {
 		return params, warnings
 	}
-
+	if val, ok := dsParams[StrategyConfigUsePeering]; ok {
+		if val == "true" {
```

Comment:
Nitpick: this should probably ignore the value, and consider it always true if the Param exists, to be consistent with the behavior of other boolean params